WordPress/twentytwentyone
# Fix 398 fix site header alignment

Fixes #398

## Summary
Center vertically the navigation when there is a custom logo.

## Relevant technical choices:
No

## Test instructions

This PR can be tested by following these steps:

1. Add a primary navigation menu to the site
2. Add a custom logo to the site
3. On frontend, the navigation menu should be vertically aligned in the site `header`

## Quality assurance
